source/assets/stylesheets/style.css

```diff
@@ -126,6 +126,7 @@ body {
 	min-height: 380px;
 	float: right;
 	position: relative;
+  border: 10px solid #333;
```

Comment:
you mean you are not seeing any issues in your editor locally?  (in that case you can just merge and we can fix your setup together soon)